### PR TITLE
More fixes for windows. #69

### PR DESCRIPTION
**Pull request by Acelya-9028**

### CODE REVIEW

**Comment by Acelya-9028** [comment]
Incredible xD

File: lua/gitsigns/types.lua [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
```
Comment (lewis6991) [inline lua/gitsigns/types.lua:1]: This file isn't necessary, can you remove it?

File: teal/gitsigns/git.tl [hunk 2]
```diff
@@ -218,7 +217,9 @@ function M.run_diff(staged: string, text: {string}, diff_algo: string): cb_funct
         end
       end,
       on_stderr = function(_, line: string)
-        print('error: '..line, 'NA', 'run_diff')
+           vim.schedule(function()
```
Comment (lewis6991) [inline teal/gitsigns/git.tl:220]: Can you add a comment saying this is needed to stop windows crashing.

**Review by lewis6991 — CHANGES_REQUESTED** [review]

File: lua/gitsigns/types.lua (hunk 1 above)
Comment (Acelya-9028) [reply lua/gitsigns/types.lua]: Done

File: teal/gitsigns/git.tl (hunk 2 above)
Comment (Acelya-9028) [reply teal/gitsigns/git.tl:221]: Done

**Comment by lewis6991** [comment]
Many thanks!

**Comment by Acelya-9028** [comment]
Thank you very much!

**Pull request merged**